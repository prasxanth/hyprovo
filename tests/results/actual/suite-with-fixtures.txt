2021-11-24 13:52:30.509 - INFO - Pass ✓... (suite-with-fixtures / test-mul-variable): mul pass
2021-11-24 13:52:30.511 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
2021-11-24 13:52:30.516 - INFO - Pass ✓... (suite-with-fixtures / test-div-variable): div pass
2021-11-24 13:52:30.518 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 1.0
2021-11-24 13:52:30.523 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul pass 
                                    Expected: 1
                                    Got: 6
2021-11-24 13:52:30.525 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 6
2021-11-24 13:52:30.530 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div pass 
                                    Expected: 1
                                    Got: 1.5
2021-11-24 13:52:30.532 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 0.6666666666666666
