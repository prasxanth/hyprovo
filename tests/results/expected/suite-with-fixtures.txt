2021-11-24 12:39:44.726 - INFO - Pass ✓... (suite-with-fixtures / test-mul-variable): mul pass
2021-11-24 12:39:44.730 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
2021-11-24 12:39:44.735 - INFO - Pass ✓... (suite-with-fixtures / test-div-variable): div pass
2021-11-24 12:39:44.738 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 1.0
2021-11-24 12:39:44.743 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul pass 
                                    Expected: 1
                                    Got: 6
2021-11-24 12:39:44.746 - ERROR - Fail ✗... (suite-with-fixtures / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 6
2021-11-24 12:39:44.751 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div pass 
                                    Expected: 1
                                    Got: 1.5
2021-11-24 12:39:44.754 - ERROR - Fail ✗... (suite-with-fixtures / test-div-variable): div fail 
                                    Expected: 4
                                    Got: 0.6666666666666666
